projects/spectator/schematics/src/spectator/index.ts

```diff
@@ -101,4 +97,8 @@ function _ensurePath(tree: Tree, options: any) {
     const projectDirName = project.projectType === 'application' ? 'app' : 'lib';
     options.path = `${root}${projectDirName}`;
   }
+
```

Comment:
else?

Reply:
It will not work as intended with `else`, because the case where `options.path` is defined will not be handled. We want to handle the path provided with the name in any case.